userinterface/cameragimbalui.cpp

```diff
@@ -84,6 +31,60 @@ void CameraGimbalUI::setGimbal(const QSharedPointer<Gimbal> gimbal)
         ui->gimbalStatusLabel->setText("Gimbal found.");
         ui->gimbalControlGroup->setEnabled(true);
         ui->cameraControlGroup->setEnabled(true);
+
+        if (!QGamepadManager::instance()->connectedGamepads().isEmpty() &&
+                QMessageBox::question(this, "CameraGimbalUI: Gamepad detected", "A gamepad was detected. Should it be used for controlling the vehicle's gimbal?") == QMessageBox::Yes) {
```

Comment:
I am getting a Segmentation fault error due to QMessageBox::question. Perhaps the text is too big for the popup window?